pkg/cmd/server/kubernetes/master_config.go

```diff
@@ -112,6 +113,17 @@ func BuildDefaultAPIServer(options configapi.MasterConfig) (*apiserveroptions.Se
 	server.GenericServerRunOptions.TLSCertFile = options.ServingInfo.ServerCert.CertFile
 	server.GenericServerRunOptions.TLSPrivateKeyFile = options.ServingInfo.ServerCert.KeyFile
 	server.GenericServerRunOptions.ClientCAFile = options.ServingInfo.ClientCA
+
+	// TODO this is a terrible hack that should be removed in 1.6
```

Comment:
@sttts we rewired this code in kube 1.6, so the rebase should barf here.  The pick is awful, so this only lasts through the rebase.